### PR TITLE
{bio}[intel/2019a] WHAM v2.0.9.1

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# No valid HTTPS - Let'sEncrypt Cert expired loong time ago
+homepage = 'http://membrane.urmc.rochester.edu/?page_id=126'
+description = """An implementation of WHAM: 
+the Weighted Histogram Analysis Method"""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+# No valid https
+source_urls = ['http://membrane.urmc.rochester.edu/sites/default/files/wham/']
+sources = ['%(namelower)s-release-2.0.9.1.tgz']
```
Comment (Micket) [inline easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a.eb:20]:
```suggestion
sources = ['%(namelower)s-release-%(version)s.tgz']
```

File: easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a-kj_mol.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+# No valid HTTPS - Let'sEncrypt Cert expired loong time ago
+homepage = 'http://membrane.urmc.rochester.edu/?page_id=126'
+description = """An implementation of WHAM: 
+the Weighted Histogram Analysis Method"""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+# No valid https
+source_urls = ['http://membrane.urmc.rochester.edu/sites/default/files/wham/']
+sources = ['%(namelower)s-release-2.0.9.1.tgz']
```
Comment (Micket) [inline easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a-kj_mol.eb:21]:
```suggestion
sources = ['%(namelower)s-release-%(version)s.tgz']
```

File: easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+checksums = ['db790ff82742527356de91af7ff95a2d7f77cc554b15596552a85130edec87e7']
+
+# might be unnecessary
```
Comment (Micket) [inline easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a.eb:23]: I think we can drop the comment? 
```suggestion
```

File: easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a-kj_mol.eb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+patches = ['%(name)s-%(version)s-change_units.patch']
+checksums = [
+    'db790ff82742527356de91af7ff95a2d7f77cc554b15596552a85130edec87e7',  # wham-release-2.0.9.1.tgz
+    'c670e99095de8c048dfb18ee33764384cdff1c1388c926183c58e83eb2d6c04f',  # WHAM-2.0.9.1-change_units.patch
+]
+
+# might be unnecessary
```
Comment (Micket) [inline easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a-kj_mol.eb:28]:
```suggestion
```

File: easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a.eb (hunk 3 above)
Comment (Darkless012) [reply easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a.eb]: dropping make clean, easybuild should create new build dir each run.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/9d91510b50f811c69ac06c5caf491275 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @Darkless012!

**Pull request merged**